src/ui/battle-info.ts

```diff
@@ -17,7 +17,7 @@ import { WindowVariant } from "#enums/window-variant";
 import i18next from "i18next";
 import { ExpGainsSpeed } from "#enums/exp-gains-speed";
 import { settings } from "#app/system/settings/settings-manager";
-import { CANVAS_SCALE, GAME_WIDTH } from "#app/ui-constants";
+import { CANVAS_SCALE } from "#app/ui-constants";
```

Comment:
```suggestion
import { CANVAS_SCALE, GAME_WIDTH } from "#app/ui-constants";
```
merge issue seems like